docs/source/rest.md

```diff
@@ -24,16 +24,28 @@ Hub.
 
 ## Create an API token
 
-To send requests using JupyterHub API, you must pass an API token with the
-request. You can create a token for an individual user using the following
-command:
+To send requests using JupyterHub API, you must pass an API token with
+the request.
 
-    jupyterhub token USERNAME
+As of [version 0.6.0](./changelog.html), the preferred way of
+generating an API token is:
+
+```bash
+openssl rand -hex 32
+```
+
+Alternatively, use the `jupyterhub token` (*deprecated in version 0.8.0*)
```

Comment:
`jupyterhub token` is not deprecated. It does two things:

1. generate a random string
2. register it for the given user with the database

The `openssl` command only generates a *potential* token, but it still must be added to the db via the `.api_tokens` mechanism. It's totally fine to recommend using `jupyterhub token`.

It would probably also be appropriate to add a mention here of the new Token request page in 0.8.